gglasso/solver/single_admm_solver.py

```diff
@@ -7,154 +7,199 @@
 from scipy.sparse.csgraph import connected_components
 from scipy.linalg import block_diag
 
-
 from .ggl_helper import prox_od_1norm, phiplus, prox_rank_norm
 
 
-def ADMM_SGL(S, lambda1, Omega_0 , Theta_0 = np.array([]), X_0 = np.array([]), \
-             eps_admm = 1e-5 , rho= 1., max_iter = 1000, verbose = False, measure = False, latent = False, mu1 = None):
+def ADMM_stopping_criterion(Omega, Theta, Theta_t_1, L, X, S, lambda1, eps_rel = 1e-3, eps_abs = 1e-3, latent=False, mu1=None):
```

Comment:
`eps_rel` and `eps_abs` have to be specifiable by the user, so they need to be an argument of `ADMM_SGL`